Source/ARTRest.m

```diff
@@ -154,7 +154,11 @@ - (void)executeRequest:(NSMutableURLRequest *)request completion:(void (^)(NSHTT
         }
         if (retries < _options.httpMaxRetryCount && [self shouldRetryWithFallback:request response:response error:error]) {
             if (!blockFallbacks && [request.URL.host isEqualToString:(_prioritizedHost ? _prioritizedHost : [ARTDefault restHost])]) {
```

Comment:
Thanks @tcard, however @CezaryKopacz's scope was only to look at `RSC15a` for now, so that's a valid comment, but will be addressed in a separate PR.